Altis_Life.Altis/config/Config_Clothing.hpp

```diff
@@ -261,26 +279,36 @@ class Clothing {
         conditions = "license_civ_rebel";
         side = "civ";
         uniforms[] = {
-            { "NONE", "Remove Uniform", 0, "" },
+            { "NONE", $STR_C_Remove_uniforms, 0, "" },
             { "U_IG_Guerilla1_1", "", 5000, "" },
             { "U_I_G_Story_Protagonist_F", "", 7500, "" },
             { "U_I_G_resistanceLeader_F", "", 11500, "" },
-            { "U_IG_leader", "Guerilla Leader", 15340, "" },
+            { "U_IG_leader", "", 15340, "" },
             { "U_O_PilotCoveralls", "", 15610, "" },
             { "U_O_SpecopsUniform_ocamo", "", 17500, "" },
-            { "U_O_GhillieSuit", "", 50000, "" }
+            { "U_O_GhillieSuit", "", 50000, "" },
+            { "U_B_T_Soldier_F", "", 50000, "" }, //Apex DLC
+            { "U_I_C_Soldier_Bandit_1_F", "", 50000, "" }, //Apex DLC
+            { "U_I_C_Soldier_Bandit_2_F", "", 51000, "" }, //Apex DLC
+            { "U_I_C_Soldier_Bandit_3_F", "", 50032, "" }, //Apex DLC
+            { "U_I_C_Soldier_Bandit_4_F", "", 50020, "" }, //Apex DLC
+            { "U_I_C_Soldier_Bandit_5_F", "", 50020, "" }, //Apex DLC
+            { "U_I_C_Soldier_Camo_F", "", 52000, "" } //Apex DLC
         };
         headgear[] = {
-            { "NONE", "Remove Hat", 0, "" },
+            { "NONE", $STR_C_Remove_headgear, 0, "" },
             { "H_Bandanna_camo", "", 650, "" },
             { "H_ShemagOpen_khk", "", 800, "" },
             { "H_ShemagOpen_tan", "", 850, "" },
             { "H_Shemag_olive", "", 850, "" },
             { "H_MilCap_oucamo", "", 1200, "" },
-            { "H_HelmetO_ocamo", "", 2500, "" }
+            { "H_MilCap_gen_F", "", 1200, "" }, //Apex DLC
+            { "H_MilCap_tna_F", "", 1200, "" }, //Apex DLC
+            { "H_HelmetO_ocamo", "", 2500, "" },
+            { "H_HelmetB_Enh_tna_F", "", 8000, "" } //Apex DLC
```

Comment:
Same model as the [`H_HelmetB_plain_mcamo`](https://community.bistudio.com/wiki/Arma_3_CfgWeapons_Equipment#H_HelmetB_plain_mcamo). Should be given to police.